crates/biome_js_analyze/src/options.rs

```diff
@@ -36,20 +34,36 @@ pub enum PossibleOptions {
     NamingConvention(#[bpaf(external(naming_convention_options), hide)] NamingConventionOptions),
     /// Options for `noRestrictedGlobals` rule
     RestrictedGlobals(#[bpaf(external(restricted_globals_options), hide)] RestrictedGlobalsOptions),
-    /// No options available
-    #[default]
-    NoOptions,
 }
 
+// Required by [Bpaf].
 impl FromStr for PossibleOptions {
     type Err = ();
 
     fn from_str(_s: &str) -> Result<Self, Self::Err> {
-        Ok(Self::NoOptions)
+        Ok(Self::Complexity(ComplexityOptions::default()))
     }
 }
 
 impl PossibleOptions {
+    pub fn try_from_rule_name(rule_name: &str) -> Option<Self> {
```

Comment:
In Rust world, functions that start with `try_*` usually return a `Result`. This comes from `try_from`/`try_into`.

Since this function doesn't return an error and it creates `Self`, we could call it `new_from_rule_name`